src/qt/locale/bitcoin_zh_HK.ts

```diff
@@ -390,7 +390,7 @@
     <name>Intro</name>
     <message>
         <source>Bitcoin</source>
```

Comment:
This should probably be replaced with Dogecoin if converting converting Bitcoin to Dogecoin for the page